<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Node_Float</class>
 <widget class="QWidget" name="Node_Float">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>125</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>175</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Node_Float</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(14, 93, 112);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_name">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(177, 177, 177);</string>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_delete">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"> QPushButton#btn_delete {
     background-color :rgb(181,109,30);
 }
 QPushButton#btn_delete:hover {
     background-color: rgb(14, 93, 112);
 }</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>179</width>
          <height>74</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_slider_value">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_value">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
background: rgb(97,97,97);
height: 10px;
border-radius: 4px;
}
QSlider::handle:horizontal {
background: rgb(38,85,123);
width: 13px;
margin-top: -10px;
margin-bottom: -10px;
border-radius: 4px;
}</string>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BinaryTree.qrc"/>
 </resources>
 <connections/>
</ui>
